src/widgets/programs/programs-overview.component.tsx

```diff
@@ -4,7 +4,10 @@ import SummaryCard from "../../ui-components/cards/summary-card.component";
 import SummaryCardRow from "../../ui-components/cards/summary-card-row.component";
 import SummaryCardRowContent from "../../ui-components/cards/summary-card-row-content.component";
 import EmptyState from "../../ui-components/empty-state/empty-state.component";
-import { fetchEnrolledPrograms } from "./programs.resource";
+import {
+  fetchEnrolledPrograms,
```

Comment:
Is this still needed?

Reply:
Yes. The ProgramsDetailedSummary displays all programs and would therefore still need this.

Reply:
I meant "is the import still needed", not "is the function at all still needed" - I think it was removed now. Thanks!